### PR TITLE
CLDR-14386 Modernize List Users; starting to get mail working...

### PR DESCRIPTION
**Pull request by btangmu**
-Mostly changes to cldrAccount.js and UserList.java

-Add new js/src to pretty.sh; js/src reformatted accordingly

-Fix outdated references to SurveyAjax.JSONWriter in AdminAjax.jsp

-Fix minor bugs discovered in cldrAdmin.js, cldrLoad.js, cldrSurvey.js

##### Checklist

- [x] Issue filed: https://unicode-org.atlassian.net/browse/CLDR-14386
- [x] Updated PR title and link in previous line to include Issue number

### CODE REVIEW

File: tools/cldr-apps/js/src/helloer.js [hunk 1]
```diff
@@ -1,14 +1,13 @@
...
-	static getHello() {
-		return ['Hello', 'from', 'webpack', 'module'].join(' ');
-	}
+  /**
+   * Load the greeting
+   */
+  static getHello() {
+    return ["Hello", "from", "webpack", "module"].join(" ");
+  }
 }
```
Comment (btangmu) [inline tools/cldr-apps/js/src/helloer.js:11]: Only formatting per npx prettier

File: tools/cldr-apps/js/src/index.js [hunk 2]
```diff
@@ -1,17 +1,16 @@
-import './style.css';
+import "./style.css";
```
Comment (btangmu) [inline tools/cldr-apps/js/src/index.js:1]: Only formatting per npx prettier

File: tools/cldr-apps/src/main/webapp/js/new/cldrAdmin.js [hunk 3]
```diff
@@ -177,7 +177,7 @@ const cldrAdmin = (function () {
           cldrDom.createChunk("Session: " + sess, "span", "adminUserSession")
         );
         if (cs.user) {
-          user.appendChild(cldrListUsers.createUser(cs.user));
+          user.appendChild(cldrAdmin.createUser(cs.user));
         } else {
```
Comment (btangmu) [inline tools/cldr-apps/src/main/webapp/js/new/cldrAdmin.js:181]: almost everything moved from cldrListUsers to cldrAdmin in the previous pr, but I didn't remember to change this one then

webpack actually pointed out the error

File: tools/cldr-apps/src/main/webapp/js/new/cldrLoad.js [hunk 4]
```diff
@@ -1077,36 +1077,6 @@ const cldrLoad = (function () {
     );
   }
 
-  function flipToOtherDiv(div) {
-    flipper.flipTo(pages.other, div);
```
Comment (btangmu) [inline tools/cldr-apps/src/main/webapp/js/new/cldrLoad.js:1081]: some code was duplicated, probably I did copy-paste when I intended cut-paste -- webpack noticed this too

File: tools/cldr-apps/src/main/webapp/js/new/cldrSurvey.js [hunk 5]
```diff
@@ -1366,22 +1366,6 @@ const cldrSurvey = (function () {
     surveyUserCov = cov;
   }
 
-  function getSurveyOrgCov() {
-    return surveyOrgCov;
```
Comment (btangmu) [inline tools/cldr-apps/src/main/webapp/js/new/cldrSurvey.js:1370]: duplication again, discovered by webpack again

**Comment by btangmu** [comment]
This is still very much work in progress, good to commit anyway since this is for non-Dojo, won't affect production yet

**Comment by srl295** [comment]
@btangmu seems to have merge conflicts

File: tools/cldr-apps/src/main/webapp/js/new/cldrSurvey.js (hunk 5 above)
Comment (srl295) [reply tools/cldr-apps/src/main/webapp/js/new/cldrSurvey.js:1370]: very nice.  

basically, the module system is enforcing some discipline in the namespace.

**Review by srl295 — APPROVED** [review]

**Comment by srl295** [comment]
@btangmu nevermind about the merge conflicts, must have been an error.

**Pull request merged**